addrparser/locales/fi.py

```diff
@@ -54,7 +54,7 @@
     [
         (porras(ENTRANCE) + Opt(Suppress(Space()) + huoneisto(APARTMENT_NUMBER))),
         (
-            Suppress(one_of(["as.", "bst."]))
```

Comment:
Maybe a bit nitpicking but adding the period as Optional would be the most logical way to implement this.
```python
Suppress(one_of(["as", "bst"])+Opt("."))
```
This tough is so small addition so your original proposal is also a valid.